src/core/datastore/utils/utils.js

```diff
@@ -0,0 +1,38 @@
+import { Metadata } from '../../metadata';
+import { Client } from '../../client';
+
+export function buildCollectionUrl(collectionName, id, restAction) {
+  let result = `appdata/${Client.sharedInstance().appKey}/${collectionName}`;
+  if (id) {
+    result += `/${id}`;
```

Comment:
We should use `url-join` which is already a dependency of the repo.

Reply:
Okay, sure 👍

Reply:
This is done in a later PR.